test/index.test.js

```diff
@@ -1,37 +1,123 @@
 const {expect} = require('chai');
 const {options} = require('./initOptions');
 
```

Comment:
I'd like to consolidate our testing frameworks, we're using many different solutions for testing.
My personal opinion is Jest, but if a discussion is needed to agree on a tech stack we can do that